src/bin/mcrt.rs

```diff
@@ -84,6 +84,14 @@ fn main() {
         .expect("Failed to link attribute to surfaces.");
     report!(surfs, "surfaces");
 
+    /*
+     * Create a boundary for the simulation with boundary coneditions. 
```

Comment:
*conditions typo

Reply:
This has been fixed in https://github.com/aetherus-wg/Aetherus/pull/56/commits/59c4f43ab36666c64d2eda2ad01ed217b2f07ea4.